src/function/FunctionEditVars.ts

```diff
@@ -277,3 +279,31 @@ export function initElements() {
   }
   return ids;
 }
+
+export function addToCount(vocabulary: string, ...terms: string[]) {
```

Comment:
The following two functions are virtually the same, except for the arithmetic operation. I'd probably opt for some kind of abstraction, e.g.

```typescript
function changeCount(vocabulary: string, terms: string[], op: (count: number) => number) {
  ...
  const vocabularyCount = WorkspaceVocabularies[vocabulary].count;
  ...
  count[Representation.FULL] = op(count[Representation.FULL]);
  ...
  count[Representation.COMPACT] = op(count[Representation.COMPACT]);
  ...
}

export function addToCount(vocabulary: string, ...terms: string[]) {
  return changeCount(vocabulary, terms, c => c + 1);
}
```

Reply:
Thanks for the idea! 4095212 implements it.